code/Test_definitions/geofencing-subscriptions.feature

```diff
@@ -150,79 +203,165 @@ Feature: Camara Geofencing Subscriptions API, vwip - Operations on subscriptions
     And an event notification of the subscribed type is received on callback-url
     And notification body complies with the OAS schema at "#/components/schemas/CloudEvent"
 
-########################### Error response scenarios ############################################
-  @geofencing_subscriptions_12_create_geofencing_subscription_for_a_device_with_invalid_parameter
-  Scenario:  Create geofencing subscription with invalid parameter
-    Given the request body is not compliant with the schema "/components/schemas/SubscriptionRequest"
-    When the  request "createGeofencingSubscription" is sent
-    Then the response code is 400
+  # Error scenarios for management of input parameter device
```

Comment:
These scenarios need to be updated for subscription request as there is no `$.device` there but "$.config.subscriptionDetail.device"